WordPress/src/main/java/org/wordpress/android/ui/posts/FeaturedImageHelper.kt

```diff
@@ -137,6 +149,24 @@ internal class FeaturedImageHelper @Inject constructor(
         return FeaturedImageData(FeaturedImageState.REMOTE_IMAGE_LOADING, photonUrl)
     }
 
+    fun trackFeaturedImageEvent(
+        stat: Stat,
```

Comment:
I'm a bit confused why we decided to change this parameter from `TrackableEvent` to `Stat`. Can you please elaborate?

It seems `TrackableEvent` enum is now pointless since we lost the type safety it brought us. We might want to consider changing this parameter back to `TrackableEvent` and calling `getLabel()` inside this method. Wdyt?

Reply:
My bad :neutral_face:. Fixed in 20a7fe3.